### PR TITLE
Feat/database

### PR DESCRIPTION
**Pull request by FinnHuelsbusch**
all classes are implemented as needed. The Controllers and repositorys will be added when needed.

### CODE REVIEW

File: backend/src/main/java/com/dhbw/tutorsystem/course/Course.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Setter
+    private int year; 
+
+    @Getter
+    @Setter
+    private String 	specialization; 
+
+    @Getter
+    @Setter
+    private String 	courseOfDegree;
```
Comment (robinbaeurle) [inline backend/src/main/java/com/dhbw/tutorsystem/course/Course.java]: Remove tabs between "String" and the attribute name

File: backend/src/main/java/com/dhbw/tutorsystem/module/Module.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Module {
```
Comment (robinbaeurle) [inline backend/src/main/java/com/dhbw/tutorsystem/module/Module.java:12]: Why is the `@Table` annotation not necessary here?

Comment (FinnHuelsbusch) [reply backend/src/main/java/com/dhbw/tutorsystem/module/Module.java:12]: @table isn't needed anymore due to new application.properties

File: backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="offer") 
+public class Offer {
```
Comment (robinbaeurle) [inline backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java]: Might want to consider specifying `Offer` and `Request` more precisely, such as `TutorialOffer`and `TutorialRequest`

Comment (timkolloch) [reply backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java]: Sounds reasonable

**Review by timkolloch — DISMISSED** [review]

File: backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java (hunk 3 above)
Comment (FinnHuelsbusch) [reply backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java]: Was solved in 09524b86d247fef36caf37bb4e3d0e135a82737b

File: backend/src/main/java/com/dhbw/tutorsystem/course/Course.java (hunk 1 above)
Comment (FinnHuelsbusch) [reply backend/src/main/java/com/dhbw/tutorsystem/course/Course.java]: was solved in [09524b8](https://github.com/FinnHuelsbusch/tutorensystem/pull/12/commits/09524b86d247fef36caf37bb4e3d0e135a82737b)

**Review by robinbaeurle — APPROVED** [review]

**Pull request merged**